README.md

```diff
@@ -2,7 +2,7 @@
   astro
 </h1>
   <h3 align="center">
-  workflows made easy<br><br>
+  Airflow workflows made easy<br><br>
```

Comment:
The `astro-python` SDK is focused on making Airflow workflows easier - but the overall repo is more ambitious.
@raphaelauv could you update the [python-sdk/README.md](https://github.com/astronomer/astro-sdk/tree/main/python-sdk) instead of the root `README.md`? The content of the root README.md is under review.